release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,195 +1,223 @@
 Bug Triage Release Team Role
```

Comment:
Toward the goal of simplifying and streamlining, a Table Of Contents is a handy trick to show up front that an extensive doc is coherent across its whole. I think a Table Of Contents here would be useful both for that shortened summary bullet list description of the role and task flow across time as well as quick links to the relevant info for what the reader may know to be the current phase.  Most of the time people probably don't read this document straight through beginning to end, but rather jump to the current thing for the queries/urls/text reminders of the moment.

Reply:
thought about it but forgot, adding it

Reply:
added TOC